api/agentlocalpb/agentlocal.proto

```diff
@@ -57,6 +57,12 @@ message StatusResponse {
   repeated AgentInfo agents_info = 4;
 }
 
+// RestartRequest describes request to restart pmm-agent.
```

Comment:
Let's call it reload. We do not actually restart pmm-agent process. Emphasis should be on configuration reloading.